.github/workflows/continuous-integration-karma-dashboard.yml

```diff
@@ -0,0 +1,48 @@
+name: Dashboard Integration Tests
+
+on:
+  push:
+    branches:
+      - master
+      - release/*
+  pull_request:
+
+jobs:
+  karma:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Read .nvmrc
+        run: echo "##[set-output name=NVMRC;]$(cat .nvmrc)"
+        id: nvm
+
+      - name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ steps.nvm.outputs.NVMRC }}
+
+      - name: Setup npm cache
+        uses: pat-s/always-upload-cache@v1.1.4
+        with:
+          path: node_modules
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-npm-${{ env.cache-name }}-
+            ${{ runner.os }}-npm-
+            ${{ runner.os }}-
+
+      - name: Install dependencies
+        run: npm ci
+        env:
+          CI: true
+
+      - name: Run dashboard integration tests
+        run: npm run test:karma:dashboard -- --headless --viewport=1600:1000 --coverage
+
+      - name: Upload code coverage report
+        uses: codecov/codecov-action@v1
+        with:
+          file: build/logs/karma-coverage/lcov.info
```

Comment:
It'd be safer, imho, to separate this log file. E.g. `build/logs/karma-coverage/(editor|dashboard)/lcov.info`

Reply:
I mean here `s/editor/edit-story`.